web/src/main/java/com/marklogic/hub/web/service/EntityManagerService.java

```diff
@@ -142,9 +143,12 @@ public EntityModel saveEntity(EntityModel entity) throws IOException {
     }
 
     public void deleteEntity(String entity) throws IOException {
-        File entitiesFile = hubConfig.getHubEntitiesDir().resolve(entity + ENTITY_FILE_EXTENSION).toFile();
+        String entityFileName = entity + ENTITY_FILE_EXTENSION;
+        File entitiesFile = hubConfig.getHubEntitiesDir().resolve(entityFileName).toFile();
         if (entitiesFile.exists()) {
             em.deleteEntity(entity);
+            dataHubService.deleteDocument("/entities/" + entityFileName, DatabaseKind.STAGING);
+            dataHubService.deleteDocument("/entities/" + entityFileName, DatabaseKind.FINAL);
```

Comment:
I don't know what our rules of thumb are for logging yet, but I think there's value in info-level logging of each URI being deleted (that would be an argument for a general-purpose "delete document" method, but I'd still hold off on that until I have another use case for it) - e.g. "Deleting URI (uri) from (database kind) database".

Reply:
Yes, the info level logging makes sense. I will add that.